screen/accounts/src/test/java/com/ivy/accounts/AccountsTabPaparazziTest.kt

```diff
@@ -13,9 +13,16 @@ class AccountsTabPaparazziTest(
     private val theme: PaparazziTheme,
 ) : PaparazziScreenshotTest() {
     @Test
-    fun `snapshot accountTab composable`() {
+    fun `snapshot accountTab nonCompact composable`() {
```

Comment:
You need to record Paparazzi snapshots, see the CI Troubleshooting guide

Reply:
Are you referring to Paparazzi's [readme](https://github.com/cashapp/paparazzi) or something else? If something else, it'd be really helpful if you provide me the link of  CI Troubleshooting guide . Thanks in advance.

Reply:
![Capture](https://github.com/user-attachments/assets/1ab4e704-8cd3-408e-a82b-37cfd1971cbe)

Failures are from `FakeTransactionDaoTest`. Are they flakey or caused by changes of this PR?

Reply:
Looks flakiness, do retry fix them? Lately they worked on the CI

Reply:
Feel free to delete those

Reply:
Deleted those 2 tests!